app/scripts/modules/kubernetes/instance/details/details.controller.js

```diff
@@ -29,6 +29,10 @@ module.exports = angular.module('spinnaker.instance.detail.kubernetes.controller
       standalone: app.isStandalone,
     };
 
+    this.uiLink = function uiLink() {
```

Comment:
The scope isn't computed until a little while after the controller is initialized, so for the sake of handling each of the three resources identically I made this a function. Is there a neater way to do it?